llvm/test/tools/llvm-mca/RISCV/SiFiveP600/zvbc.s

```diff
@@ -0,0 +1,176 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+# RUN: llvm-mca -mtriple=riscv64 -mcpu=sifive-p670 -iterations=1 < %s | FileCheck %s
+
+vsetvli zero, zero, e32, mf8, tu, mu
```

Comment:
Same comment as zvbb.s

Reply:
CLMUL only supports SEW=64

Reply:
My bad, I read `SEW is any value other than 64` as that is what was required, instead of what was reserved.